styles.css

```diff
@@ -672,3 +714,638 @@ button:hover {
+  }
+
+  /* .phone {
+    display: none;
+  } */
+  header,
+  section {
+    justify-content: center;
+    align-items: center;
+    min-width: (90%, 875px);
+  }
+
+  .AppBar {
+    height: 72px;
+    width: 100%;
+  }
+
+  .ToolBar {
+    width: 100%;
+  }
+
+  .nav-menu {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 100px;
+    margin-right: 120px;
+    margin-top: -20px;
+  }
+
+  .Logo {
+    margin-left: 120px;
+  }
+
+  .nav-menu li a {
+    align-self: flex-end;
+  }
+
+  .headline {
+    width: 100%;
+    display: flex;
+    gap: 1rem;
+    height: 850px;
+  }
+
+  .image-headline {
+    background: url("images/Illustration-header.png");
+    background-position: right;
+    background-size: cover;
+    background-repeat: no repeat;
+    width: 1000.13px;
+    height: 769.27px;
+    top: -140px;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: row-reverse;
+    padding-top: 250px;
+  }
+
+  .split {
+    width: 500px;
+  }
+
+  .SocialMedia {
+    flex-direction: column;
+    padding-top: 120px;
+    margin-left: 40px;
+  }
+
+  .discription p {
+    width: 400px;
+  }
+
+  .Works {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-left: 5%;
+    margin-right: 5%;
+  }
+
+  .RecentWorks {
+    flex-direction: row;
+    width: 1300px;
+  }
+
+  .RecentWorks h2 {
+    width: 40%;
+    text-align: center;
+
+    /* margin-left: 2rem; */
+  }
+
+  #line {
+    width: 60%;
+    margin-left: -78px;
+  }
+
+  .MultiPostStories {
+    flex-direction: row;
+    width: 100%;
+    margin-top: 90px;
+    margin-bottom: 20px;
+  }
+
+  .MultiPostStories img {
+    width: 684px;
+    height: 448px;
+  }
+
+  .work1 h4 {
+    padding-top: 120px;
+  }
+
+  .work1 p {
+    padding-top: 40px;
+    margin: 20px 0;
+  }
+
+  .category {
+    padding-top: 40px;
+  }
+
+  #text {
+    display: none;
+  }
+
+  .split2 {
+    align-self: flex-start;
+    margin-top: -25px;
+  }
+
+  .languageBoxes {
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
+
+  .cards {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 10px;
+    margin-top: 50px;
+  }
+
+  #w1 {
+    background-image: url("/images/desktop-Card-Works.png") !important;
+    background-repeat: no-repeat;
+    border: no border;
+    height: 435px;
+    width: 369px;
+  }
+
+  #w2 {
+    background-image: url("/images/desktop-data-dashbord-bg.png") !important;
+    background-repeat: no-repeat;
+    border: no border;
+    height: 435px;
+    width: 369px;
+  }
+
+  #w3 {
+    background-image: url("/images/desktop-website-portfolio-bg.png");
+    background-repeat: no-repeat;
+    height: 435px;
+    width: 369px;
+    border: no border;
+  }
+
+  #w4 {
+    background-image: url("/images/Img desktop@2x.png") !important;
+    background-repeat: no-repeat;
+    height: 435px;
+    width: 369px;
+    margin-top: 20px;
+  }
+
```

Comment:
- I think the `margin-top: 20px;` added to your fourth card has made it look a bit bigger/longer than the other cards

Here 👇🏽 
![cardss](https://user-images.githubusercontent.com/99246359/200112842-85f5a92b-2da9-457e-9a23-7a8b9e14c77d.PNG)

- I believe if we removed the `margin-top: 20px;`, it will fit properly and all cards will be equally aligned 🍒

Reply:
thanks for that it was try and error method which resulted into forgetting that